gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/v4/impl/ApiStateServiceImpl.java

```diff
@@ -285,31 +283,50 @@ private GenericApiEntity updateLifecycle(
         LifecycleState lifecycleState,
         String username
     ) throws TechnicalException {
-        Api api = apiRepository.findById(apiId).orElseThrow(() -> new ApiNotFoundException(apiId));
+        Optional<Api> optApi = apiRepository.findById(apiId);
```

Comment:
I don't understand why we changed this part. It looks the same but with the "explicit" if. I'm not sure the new structure is better than the previous one 🙈